internal/convert/api_keys_access_list_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit
+
+package convert
+
+import (
+	"reflect"
+	"testing"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestFromWhitelistAPIKeysToAccessListAPIKeys(t *testing.T) {
+	type test struct {
+		input *atlas.WhitelistAPIKeys
+		want  *atlas.AccessListAPIKeys
+	}
+
+	tests := []test{
+		{
+			input: &atlas.WhitelistAPIKeys{
+				Results: []*atlas.WhitelistAPIKey{
+					{
+						CidrBlock:       "1",
+						Count:           1,
+						Created:         "1",
+						IPAddress:       "1",
+						LastUsed:        "1",
+						LastUsedAddress: "1",
+						Links:           nil,
+					},
+				},
+				TotalCount: 1,
+			},
+			want: &atlas.AccessListAPIKeys{
+				Results: []*atlas.AccessListAPIKey{
+					{
+						CidrBlock:       "1",
+						Count:           1,
+						Created:         "1",
+						IPAddress:       "1",
+						LastUsed:        "1",
+						LastUsedAddress: "1",
+						Links:           nil,
+					},
+				},
+				TotalCount: 1,
+			},
+		},
+		{
+			input: &atlas.WhitelistAPIKeys{
+				Results: []*atlas.WhitelistAPIKey{
+					{
+						CidrBlock:       "1",
+						Count:           1,
+						Created:         "1",
+						IPAddress:       "1",
+						LastUsed:        "1",
+						LastUsedAddress: "1",
+						Links:           nil,
+					},
+					{
+						CidrBlock:       "1",
+						Count:           1,
+						Created:         "1",
+						IPAddress:       "1",
+						LastUsed:        "1",
+						LastUsedAddress: "1",
+						Links:           nil,
+					},
+				},
+				TotalCount: 2,
+			},
+			want: &atlas.AccessListAPIKeys{
+				Results: []*atlas.AccessListAPIKey{
+					{
+						CidrBlock:       "1",
+						Count:           1,
+						Created:         "1",
+						IPAddress:       "1",
+						LastUsed:        "1",
+						LastUsedAddress: "1",
+						Links:           nil,
+					},
+					{
+						CidrBlock:       "1",
+						Count:           1,
+						Created:         "1",
+						IPAddress:       "1",
+						LastUsed:        "1",
+						LastUsedAddress: "1",
+						Links:           nil,
+					},
+				},
+				TotalCount: 2,
+			},
+		},
+	}
+
+	for _, tc := range tests {
+		got := FromWhitelistAPIKeysToAccessListAPIKeys(tc.input)
+		if !reflect.DeepEqual(tc.want, got) {
+			t.Fatalf("expected: %v, got: %v", tc.want, got)
+		}
```

Comment:
[nb] I guess this was autogenerate by idea  but you can use `assert.Equals` here